libs/application-generic/src/usecases/workflow/get-workflow-by-ids/get-workflow-by-ids.usecase.ts

```diff
@@ -41,6 +50,26 @@ export class GetWorkflowByIdsUseCase {
       });
     }
 
-    return workflowEntity;
+    const workflowPreferences = await this.getPreferences.safeExecute(
+      GetPreferencesCommand.create({
+        environmentId: command.environmentId,
+        organizationId: command.organizationId,
+        templateId: workflowEntity._id,
+      }),
+    );
+    /**
+     * @deprecated - use `userPreferences` and `defaultPreferences` instead
```

Comment:
I guess this needs to be removed after data migration. If that's the case, let's mark all the places in the code that can be removed after the data migration so that we can spot them easily.

Reply:
Yes, it does. Everywhere that needs removal is effectively marked with the presence of the `preferenceSettings` data property, that implicitly acts as the marker.